### ISSUE TITLE
I try analyze my reposiroty, but get error: `invalid Auth Token`

### ISSUE THREAD

**Issue by batazor** [step-security/secure-repo#2342]
https://agent.api.stepsecurity.io/v1/app/securerepo/isauthorized

payload: `{"repository_url":"https://github.com/shortlink-org/shortlink"}`
response: `{"type":"Invalid_Auth_Token","message":"invalid Auth Token"}`

**Comment by varunsh-coder** [comment]
Hi @batazor it might have been a transient issue. Can you please try again?

I tried you repo and was able to analyze it.

**Comment by batazor** [comment]
Yeah, it went away for me, too. I've been seeing this problem for a couple weeks, tried different browsers.

In any case, the check went through, that's good
